db/models.py

```diff
@@ -1,4 +1,9 @@
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
+from django.db.models.query_utils import select_related_descend
```

Comment:
The import of `select_related_descend` is unnecessary as it is not used in the code. Consider removing it to clean up the imports.